qualtran/bloqs/hamiltonian_simulation/hamiltonian_simulation_by_gqsp.py

```diff
@@ -180,18 +172,6 @@ def build_composite_bloq(self, bb: 'BloqBuilder', **soqs: 'SoquetT') -> Dict[str
 
         return soqs
 
-    def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
```

Comment:
what happened to this?

Reply:
We don't need a specialization anymore, it works automatically through `GeneralizedQSP`'s call graph.

Before this, there was no way to build GQSP without the exact polynomial, so I had to override it here.